lib/ClangImporter/ClangImporter.cpp

```diff
@@ -610,6 +618,12 @@ importer::getNormalInvocationArguments(
       "-DSWIFT_CLASS_EXTRA=",
     });
 
+    // Indicate that using '__attribute__((swift_attr))' with '@Sendable' and
+    // '@_nonSendable' on Clang declarations is fully supported, including the
+    // 'attribute push' pragma.
+    if (clangSupportsPragmaAttributeWithSwiftAttr())
+      invocationArgStrs.push_back( "-D__SWIFT_ATTR_SUPPORTS_SENDABLE_DECLS=1");
```

Comment:
Does Clang not have a corresponding `__has_feature` we can check in the headers? That's the more traditional way to check for Clang capabilities.

Reply:
Not for saying that a specific attribute is supported by the `attribute push` pragma, no.